core/app/models/spree/order.rb

```diff
@@ -400,7 +400,7 @@ def deliver_order_confirmation_email
 
     # Helper methods for checkout steps
     def paid?
-      payment_state == 'paid' || payment_state == 'credit_owed'
+      %w(paid credit_owed).include?(payment_state)
```

Comment:
@cbrunsdon I think we'll have to agree to disagree. Picking a date is fine, but at the moment you don't have any criteria on which to base if that date is appropriate when it comes. 

I think that's my larger issue here: There seems to be too much happening based on feels and appearances rather than on axioms and published rules. I know it's early in the project, but it's more important now than ever to establish precedent.

This PR is a perfect example of what happens without a set of published rules on a bigger, high traffic OSS project. A one line change has turned into significant conversation. Were the rules in place, a simple test, "Does this propose change follow our rules/guidelines/axioms?" could be applied. 

This is even useful in the design world, where things tend to be less scientific and more feels based, under the best practice of Design Principles.

And, while I'm thrilled that there's attempts at consensus building through the use of these core team meetings, it feels more like stonewalling from here. There's no reason to me that those same conversations that would happen behind the closed doors of a core team meeting couldn't happen in the broad daylight of a PR comment thread. Not to mention, I think that waiting for a once per week meeting adds significantly more drag and friction to getting things done quickly and well.

I'm acutely aware that the core team is trying to make sure that there's as little friction as possible so that the momentum can build for Solidus and that Solidus represents a step in the right direction. But I think the (fuzzy) lines being drawn in the sand are a dangerous and slippery slope that ends in a project like Spree.

All that said, I *know* you guys are trying to do the right thing. And, believe me when I say that I don't think anyone is doing anything but what they think is best. I just have to disagree with the decisions so far. And, I've seen them in a few PRs now, in only one short day of watching.